### ISSUE TITLE
timeout in cluster operation

### ISSUE THREAD

**Issue by weickdev** [arangodb/arangodb#3602]
## my environment running ArangoDB
I'm using the latest ArangoDB of the respective release series:
- [ ] 2.8
- [ ] 3.0
- [ ] 3.1
- [x] 3.2.5
- [ ] self-compiled devel branch

Mode:
- [x] Cluster
- [x] Single-Server

Storage-Engine:
- [ ] mmfiles
- [x] rocksdb

On this operating system:
- [ ] DCOS on
  - [ ] AWS
  - [ ] Azure
  - [ ] own infrastructure
- [ ] Linux 
  - [ ] Debian .deb
  - [ ] Ubuntu .deb
  - [ ] SUSE   .rpm
  - [x] RedHat .rpm
  - [ ] Fedora .rpm
  - [ ] Gentoo
  - [ ] docker - official docker library
  - [ ] other:
- [ ] Windows, version: 
- [ ] MacOS, version:

When I import datas through Java API, there are many timeout  exceptions during the imprting: 
```
com.arangodb.ArangoDBException: Response: 500, Error: 1457 - timeout in cluster operation
	at com.arangodb.internal.velocystream.VstCommunication.checkError(VstCommunication.java:106) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.internal.velocystream.VstCommunicationSync.execute(VstCommunicationSync.java:134) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.internal.velocystream.VstCommunicationSync.execute(VstCommunicationSync.java:46) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.internal.velocystream.VstCommunication.execute(VstCommunication.java:96) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.internal.velocystream.VstProtocol.execute(VstProtocol.java:46) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.internal.ArangoExecutorSync.execute(ArangoExecutorSync.java:58) ~[arangodb-java-driver-4.2.0.jar!/:?]
	at com.arangodb.ArangoCollection.insertDocuments(ArangoCollection.java:117) ~[arangodb-java-driver-4.2.0.jar!/:?]
```

**Comment by weickdev** [comment]
I can't make sure, if ti is the same as issue #3598

**Comment by dothebart** [comment]
Hi, 
this may be related to #3655 ; can you upgrade and revalidate whether the issue still occurs?

**Comment by dothebart** [comment]
Will close this as solved with 3.2.8
